### DISCUSSION TITLE
Boolean parameter isn't passed down correctly

### DISCUSSION THREAD

**Discussion by DarrylBrooks97** [supabase/supabase#5686] [General]
Hey everyone, 
I know the functions are still in Alpha but I just wanted to see if anyone else has had this error also.

So when creating my Postgres function to make a http_post request I realized that my first parameter here ```isFeatured```:
<img width="559" alt="Screen Shot 2022-02-26 at 10 45 30 PM" src="https://user-images.githubusercontent.com/24500251/155868831-a9b255ce-e6bb-4111-8e51-c8fbd8442908.png">
doesn't get passed down as boolean, but as text and clips only the first letter.
<img width="929" alt="Screen Shot 2022-02-26 at 10 45 18 PM" src="https://user-images.githubusercontent.com/24500251/155868900-18fa1797-cb8e-4a2e-b76c-849d9ce64a3d.png">

haven't seen a discussion about this so I thought I'd mention it for future improvement.

**Comment by steve-chavez** [comment]
Can you share the get_posts function signature?
